### PR TITLE
TRACING-3574 | Add span RED alerting docs

### PR DESCRIPTION
**Pull request by pavolloffay**
Version(s):

4.12, 4.13, 4.14, 4.15, 4.16

This PR should be merged once RHOSDT 3.1 is released.

Issue:

https://issues.redhat.com/browse/TRACING-3574

Link to docs preview:

https://69962--ocpdocs-pr.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring#distr-tracing-tempo-config-spanmetrics_distr-tracing-tempo-configuring

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by IshwarKanse** [comment]
Verified the feature and docs.

**Review by IshwarKanse — APPROVED** [review]

**Comment by max-cx** [comment]
@pavolloffay, I'm suggesting an edit in https://github.com/openshift/openshift-docs/pull/70858/files and preview https://70858--ocpdocs-pr.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring#span-red-metrics-and-alerting-rules. Could you please review it and let me know WDYT?

**Review by max-cx — APPROVED** [review]

File: modules/distr-tracing-tempo-config-spanmetrics.adoc [hunk 1]
```diff
@@ -92,3 +92,55 @@ spec:
...
+|
+* `SPAN_KIND_SERVER`
+* `SPAN_KIND_CLIENT`
+* `SPAN_KIND_PRODUCER`
+* `SPAN_KIND_CONSUMER`
+* `SPAN_KIND_INTERNAL`
+
+|===
+
+.Example PrometheusRule CR that defines an alerting rule for SLO when not serving 95% of requests within 2000ms on the front-end service
```
Comment (adellape) [inline modules/distr-tracing-tempo-config-spanmetrics.adoc:127]:
```suggestion
.Example `PrometheusRule` CR that defines an alerting rule for SLO when not serving 95% of requests within 2000ms on the front-end service
```

File: modules/distr-tracing-tempo-config-spanmetrics.adoc [hunk 2]
```diff
@@ -92,3 +92,55 @@ spec:
 ----
 <1> Enables the monitoring tab in the Jaeger console.
 <2> The service name for Thanos Querier from user-workload monitoring.
+
+== Span RED metrics and alerting rules
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply modules/distr-tracing-tempo-config-spanmetrics.adoc:96]: I will take it as out of scope for this PR. @max-cx (our docs writer) could consider fixing it in our docs.

[the remark this answers is not in the document]
Comment (max-cx) [reply modules/distr-tracing-tempo-config-spanmetrics.adoc:96]: I can go through all of the distributed files and add anchor IDs wherever they might be missing.

**Comment by kalexand-rh** [comment]
4.11 is now [EOL](https://access.redhat.com/support/policy/updates/openshift), and we are no longer updating it. Please set the versions to 4.12+.

**Comment by pavolloffay** [comment]
Closing PR @max-cx will incorporate it to his branch

File: modules/distr-tracing-tempo-config-spanmetrics.adoc [hunk 3]
```diff
@@ -92,3 +92,55 @@ spec:
...
+|`frontend`
+
+|span_name
+| Name of the operation.
+|
+* `/`
+* `/customer`
+
+|span_kind
+|Identifies the server, client, messaging, or internal operation.
```
Comment (adellape) [inline modules/distr-tracing-tempo-config-spanmetrics.adoc:117]: Can drop the punctuation on the 3 "Description" sentences

**Review by adellape — COMMENTED** [review]
Added one more comment, but LGTM for the purposes of feeding into / cherrypicking to the larger PR as described in https://github.com/openshift/openshift-docs/pull/69962#discussion_r1482738981. 👍

File: modules/distr-tracing-tempo-config-spanmetrics.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (adellape) [reply modules/distr-tracing-tempo-config-spanmetrics.adoc:96]: SGTM!

File: modules/distr-tracing-tempo-config-spanmetrics.adoc (hunk 3 above)
Comment (max-cx) [reply modules/distr-tracing-tempo-config-spanmetrics.adoc:117]: Thank you, I'll look into this.

**Pull request closed without merging**